lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,11 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.nil? or comments.last.nil?
```

Comment:
@pboling It is true that `"comments` always returns an array then the `comments.nil?` check is not needed", but in the future if the `comments` method is changed in any way by another developer, this check will be useful. Without this check, `comments.last` will throw an exception. Sorry for this, but it is better to be safe than sorry.